work_items/WACI_presentation_exchange.md

```diff
@@ -0,0 +1,64 @@
+# WACI Presentation Exchange
+
+
+## Work Item Owners
+- Orie Steele (@OR13)
+- Brent Zundel (@brentzundel)
+
+## Outcome
+**When your work item is successful, then what will have changed in the world? What
+will we be able to do that wasn't possible do before or vice versa?**
+
+We will have a standard protocol for requesting and submitting a presentation of
+a variety of credentials formats. 
+
+## Deliverables
+- **Does your work item have any deliverables? These include code projects,
+  standards-track specifications, reports, registries, summaries, blog posts,
+  and anything else that is instantiable and publishable in written form to a
+  broader audience.**
+
+We are writing a standards-track specification.
+
+- **What's the expected timeline?**
+
+We will release v0.1 for developer use by .
```

Comment:
Can we move in the preliminary timeline from IIW?

Reply:
Pasting from the notes:

```
May 1: GHP Drafting Groups First Drafts Due - 

            June 1 GHP Interoperability Blueprint Recommendations Spec complete
30 day vision
90 day vision  <- would be ideal to have something that can be tested against for this timeframe. 
180 day vision

            July 1 Test Complete

            August 1 - 10+ Implementations / Vendors passing VP Exchange Interop Tests.
	
 	October 1 - Cross Wallet Interop Exchange Tests.
```